lib/presentation/profile/profile_screen.dart

```diff
@@ -159,12 +159,25 @@ class _UserProfilePageState extends State<UserProfilePage> {
                           ),
                           const SizedBox(height: 10),
                           Center(
-                            child: Text(
-                              state.userProfile?.profile.firstName ?? 'You',
-                              style: const TextStyle(
-                                fontWeight: FontWeight.w600,
-                                fontSize: 22,
-                              ),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: [
+                                Text(
+                                  state.userProfile?.profile.firstName ?? 'Hi ',
```

Comment:
Why `Hi` ? - According to design it should stay `You`

Reply:
Yes, it is fixed, in firstname - 'You', lastname - empty